### PR TITLE
adciciona opçao dor de barriga em comer joaninha

### PR DESCRIPTION
**Pull request by matheusfacure**
"Please review this, @casey-collab"

### CODE REVIEW

File: portugues/spray/joaninha/comer/comer.md [hunk 1]
```diff
@@ -1 +1,3 @@
-Hm... o sabor parece bem exótico!
\ No newline at end of file
+Hm... o sabor parece bem exótico!
+
+[Ficou com dor de barriga?](joaninha/comer/comer/dor-de-barriga.md)
```
Comment (casey-collab) [inline portugues/spray/joaninha/comer/comer.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: portugues/spray/joaninha/comer/dor-de-barriga/dor-de-barriga.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Você vai para o banheiro, mas a dor não passa.
```
Comment (casey-collab) [inline portugues/spray/joaninha/comer/dor-de-barriga/dor-de-barriga.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by matheusfacure** [comment]
"Please review this, @casey-collab"

File: portugues/spray/joaninha/comer/comer.md (hunk 1 above)
Comment (casey-collab) [inline portugues/spray/joaninha/comer/comer.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by matheusfacure** [comment]
"Please review this, @casey-collab"

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**